.gitlab-ci.yml

```diff
@@ -194,7 +194,7 @@ Manifests:
   parallel:
     matrix:
       - RUNNER:
-          - aws/centos-stream-9-x86_64
+          - aws/rhel-9.0-ga-x86_64
```

Comment:
This won't work because our nightly rules specify 8.7 or 9.1 RHEL runners, see: https://github.com/osbuild/osbuild-composer/blob/main/.gitlab-ci.yml#L58